src/openfl/_internal/renderer/opengl/batcher/TextureData.hx

```diff
@@ -0,0 +1,16 @@
+package openfl._internal.renderer.opengl.batcher;
+
+import lime.graphics.opengl.GLTexture;
+
+class TextureData {
```

Comment:
This `Texture`/`TextureData` naming is quite poor, gotta think of something.
Basically, `TextureData` is a wrapper around GPU texture reference to store additional info used in the batcher, and `Texture` contains `TextureData` and the UV coordinates.

We use `TextureData`s to check texture change in the batcher (so there should be only one `TextureData` object per texture), and we use `Texture` for actual texture mapping (because we can have different SubBitmapDatas referencing the same texture atlas).

Reply:
PS I changed `Texture` to `QuadTextureData` - it's a less confusing name that describes actually what it is - information about texture when it comes to rendering a quad :)